src/Lib/Twig/TokenParser/Cell.php

```diff
@@ -8,16 +7,18 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
 namespace WyriHaximus\TwigView\Lib\Twig\TokenParser;
 
+use Twig\TokenParser\IncludeTokenParser;
 use Twig_Token;
```

Comment:
Is this one still used?

Reply:
Fixed this (and some PHP Iterators not being declared)